### PR TITLE
Fixed Home Return button

### PR DESCRIPTION
**Pull request by annelausf**
Home button on the debug menu does not return to home page.
Screen Shot 2021-09-23 at 8 43 26 AM

Fix:
Change the way the href in the home button is generated.
Made changes in all references to the home button in other pages.

### CODE REVIEW

**Review by sana-jawad — DISMISSED** [review]
Looks good to me!

File: pkg/sloop/webserver/webfiles/debug.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
```
Comment (venkatramreddykunta) [inline pkg/sloop/webserver/webfiles/debug.js:2]:
```suggestion
 * Copyright (c) 2021, salesforce.com, inc.
```

Do we have Copyright hook that checks Copyrights for updated and new files in the pull-requests ?

File: pkg/sloop/webserver/webfiles/debug.html [hunk 2]
```diff
@@ -9,11 +9,12 @@
     <script type="text/javascript">
         document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
     </script>
+    <script src="webfiles/debug.js"></script>
```
Comment (venkatramreddykunta) [inline pkg/sloop/webserver/webfiles/debug.html:12]: Did you do anything special for fixing the relative path issue for this file ?.

Comment (annelausf) [reply pkg/sloop/webserver/webfiles/debug.html:12]: I set the base href before making the call to reference the external js file.

**Review by venkatramreddykunta — APPROVED** [review]

File: pkg/sloop/webserver/webfiles/debugviewkey.html [hunk 3]
```diff
@@ -5,15 +5,16 @@
...
-  <head>
-      <script type="text/javascript">
-          document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
-      </script>
-      <title>Sloop Debug View Record</title>
-      <link rel='shortcut icon' type='image/x-icon' href='webfiles/favicon.ico' />
-  </head>
-<body>
-[ <a href="/">Home</a> ][ <a href="debug/">Debug Menu</a> ]<br/>
+<head>
```
Comment (kritikapradhan) [inline pkg/sloop/webserver/webfiles/debugviewkey.html:8]: nit: indentation

**Comment by annelausf** [comment]
Could not replicate issue locally, passed test when manually deployed on dev environment.

**Review by sana-jawad — APPROVED** [review]

**Pull request merged**